icons/adobeshockwaveplayer.svg

```diff
@@ -0,0 +1,2 @@
+
+<svg id="mnemonic" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 240 234"><defs><style>.cls-1,.cls-2{fill-rule:evenodd;}.cls-1{fill:url(#linear-gradient);}.cls-2{fill:url(#linear-gradient-2);}</style><linearGradient id="linear-gradient" x1="120" y1="-7.99" x2="120" y2="250.73" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#4e0000"/><stop offset="1" stop-color="#440100"/></linearGradient><linearGradient id="linear-gradient-2" x1="35" y1="117" x2="205" y2="117" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#fff"/><stop offset="1" stop-color="#e8e8e8"/></linearGradient></defs><title>shockwave_app_RGB</title><path class="cls-1" d="M0,0V234H240V0H0Z" transform="translate(0 0)"/><path class="cls-2" d="M35,55.54A6.54,6.54,0,0,1,41.54,49H41.46A6.54,6.54,0,0,1,48,55.54v56.81a6.54,6.54,0,0,1-6.54,6.54h0.08A6.54,6.54,0,0,1,35,112.35V55.54ZM55,81.93a6.53,6.53,0,0,1,6.54-6.53H61.46A6.53,6.53,0,0,1,68,81.93v56.81a6.54,6.54,0,0,1-6.54,6.54h0.08A6.54,6.54,0,0,1,55,138.74V81.93ZM75,99.78a6.54,6.54,0,0,1,6.54-6.54H81.46A6.54,6.54,0,0,1,88,99.78v56.81a6.54,6.54,0,0,1-6.54,6.54h0.08A6.54,6.54,0,0,1,75,156.59V99.78Zm19,2.51a6.53,6.53,0,0,1,6.54-6.53h-0.08a6.53,6.53,0,0,1,6.54,6.53v56.82a6.53,6.53,0,0,1-6.54,6.53h0.08A6.53,6.53,0,0,1,94,159.11V102.29Zm20-5c0-3.61,2.39-6.54,6-6.54s7,2.93,7,6.54v56.81a6.54,6.54,0,0,1-6.54,6.54h0.08a6.54,6.54,0,0,1-6.54-6.54V97.27Zm19-9.81a6.53,6.53,0,0,1,6.54-6.53h0.92A6.53,6.53,0,0,1,147,87.46v57a6.54,6.54,0,0,1-6.54,6.54h-0.92a6.54,6.54,0,0,1-6.54-6.54v-57Zm20-.25a6.54,6.54,0,0,1,6.54-6.54h-0.08A6.54,6.54,0,0,1,166,87.21V144a6.54,6.54,0,0,1-6.54,6.54h0.08A6.54,6.54,0,0,1,153,144V87.21Zm20,13.07a6.53,6.53,0,0,1,6.54-6.53h-0.08a6.53,6.53,0,0,1,6.54,6.53V157.1a6.53,6.53,0,0,1-6.54,6.53h0.08A6.53,6.53,0,0,1,173,157.1V100.28Zm19,21.37a6.54,6.54,0,0,1,6.54-6.54h-0.08a6.54,6.54,0,0,1,6.54,6.54v56.81a6.54,6.54,0,0,1-6.54,6.54h0.08a6.54,6.54,0,0,1-6.54-6.54V121.65Z" transform="translate(0 0)"/></svg>
```

Comment:
Can you remove this file as it's not needed?

Reply:
done